### ISSUE TITLE
Liquid Warning: Expected dotdot but found pipe...

### ISSUE THREAD

**Issue by shellen** [Sodaware/jekyll-archive-page#5]
On build in Jekyll I'm seeing...

```
Liquid Warning: Liquid syntax error (line 25): Expected dotdot but found pipe in "{{(page.grouped_posts[year][month] | size) }}" in archives/index.html
    Liquid Warning: Liquid syntax error (line 40): Expected dotdot but found pipe in "{{(page.grouped_posts[year][month] | size) }}" in archives/index.html
```
It still builds correctly but after some Googling, I can't figure out how to fix that. Recommendations welcome. Using jekyll 3.8.5.

**Comment by Sodaware** [comment]
Thank for the bug report! Looks like Liquid didn't like brackets in the expressions. So instead of

```html
{% assign has_posts = (page.grouped_posts[year][month] | size) %}
```

it should have been

```html
{% assign has_posts = page.grouped_posts[year][month] | size %}
```

All fixed by pull request #6.

Cheers,
Phil

**Comment by shellen** [comment]
Fixed! Thank you.
